### ISSUE TITLE
Distinguish presets for easier buildign from CLI

### ISSUE THREAD

**Issue by cassidyjames** [pigdevstudio/juan_saga#19]
In order to build the app from the CLI (instead of from within the Godot UI), you have to pass it a preset name. When all the presets have the same name, it looks like Godot gets confused and just spits out the first one (i.e. HTML5). 

Instead, I believe it makes sense to name the presets similar to the platforms that they target.

**Comment by henriiquecampos** [comment]
This is done within the Godot Project? So, for another game I can rename each preset properly :stuck_out_tongue:

**Comment by cassidyjames** [comment]
@henriiquecampos yeah I think you can set these within the Godot UI, and it's per-project.
